WordPress/gutenberg
# Separator: Remove border-bottom property

## What?
The separator block was using a top and bottom border of 1px to create the horizontal line. This change removes the bottom border and increases the `border-top` to 2px.

Fixes https://github.com/WordPress/gutenberg/issues/31424

## Why?

There was some occasional visual gap happening between the top and bottom border within the editor (was fine on the frontend). It came up in an issue regarding patterns on WooCommerce found here https://github.com/woocommerce/woocommerce-blocks/pull/11304#issuecomment-1786712514 but can be reproduced on any site.

## How?

Removing the bottom border altogether means this gap cannot appear.


## Testing Instructions





There is no real rhyme or reason that I have found to reproducing this bug, it _appears_ random but happens often enough especially when working with patterns I've found.

1. Open a post or page.
2. Insert a separator block.
3. Insert a pattern of choice.
4. Insert another separator block.
5. Keep repeating steps 2-4 and try using different combinations (e.g. different blocks, multiple separators one after another etc).
6. No double line should appear when using the separator. Check these changes on the frontend also to ensure no regressions have occured.

### Testing Instructions for Keyboard


## Screenshots or screencast 

Here is a screenshot of it happening within the editor. Another example can be found here https://github.com/woocommerce/woocommerce-blocks/pull/11304#issuecomment-1786712514

![Screenshot 2023-10-31 at 11 05 00](https://github.com/WordPress/gutenberg/assets/8639742/772d6455-6a81-45eb-accf-f61bafc1d757)
